architecture/additional_concepts/storage.adoc

```diff
@@ -254,6 +254,64 @@ take advantage of the volume's ability for ROX. Errors in the provider show up
 at runtime as mount errors.
 ====
 
+The table below lists the access modes supported by different persistent volumes.
```

Comment:
As it's involving a statement followed by what it's talking about, I'd end it with a colon instead of a period.